src/core/tests/Test_StatusHandler.py

```diff
@@ -414,6 +414,14 @@ def test_sort_packages_by_classification_and_state(self):
             self.assertEqual(installation_patches_sorted[12]["name"], "test-package-2")  # | Other              | Excluded    |
             self.assertEqual(installation_patches_sorted[13]["name"], "test-package-1")  # | Other              | NotSelected |
 
+    def test_sequence_number_changed_termination_configure_patching_only(self):
+        """ Test sequence number change for configure patching throws newer operation superseded error message """
+        self.__assert_sequence_num_changed_termination(config=Constants.CONFIGURE_PATCHING, summary=Constants.CONFIGURE_PATCHING_SUMMARY, status=Constants.STATUS_ERROR)
+
+    def test_sequence_number_changed_termination_installation_patching(self):
+        """ Test sequence number change for installation patching throws newer operation superseded error message """
```

Comment:
nit: similar to the comment above, """ Test sequence number change for installation ..... """"